docs/bigquery-usage.rst

```diff
@@ -281,7 +281,7 @@ Run a query which can be expected to complete within bounded time:
    >>> from gcloud import bigquery
    >>> client = bigquery.Client()
    >>> query = """\
-   SELECT count(*) AS age_count FROM dataset_name.person_ages
+   SELECT count(*) AS age_count FROM [project-name:dataset_name.person_ages]
```

Comment:
@tseaver I originally intended to use `table.query_id` here but figured it was overkill to construct a dataset and then a table.